setup.cfg

```diff
@@ -50,12 +49,11 @@ ignore =
     # These are forbidding certain __future__ imports. The future-import plugin
     # has errors both for having and not having them; we want to have these until
     # Sopel no longer supports Python versions that require them.
-    FI55,
+    FI58,
     # These would require future imports that are not needed any more on Sopel's
-    # oldest supported Python version (3.6).
-    FI10,FI11,FI12,FI13,FI14,FI16,FI17,
+    # oldest supported Python version (3.7).
+    FI10,FI11,FI12,FI13,FI14,FI15,FI16,FI17,
     # Ignore "annotations" future import, since it's not available before 3.7
```

Comment:
```suggestion
```

We're not ignoring this one any more. Don't know if trailing comma is allowed in this value, so you might need to remove the comma after `FI17` above.